homeassistant/components/velbus/light.py

```diff
@@ -36,11 +40,27 @@
 class VelbusLight(VelbusEntity, Light):
     """Representation of a Velbus light."""
 
+    @property
+    def name(self):
+        """Return the display name of this entity."""
+        if self._module.light_is_buttonled(self._channel):
+            return "LED " + self._module.get_name(self._channel)
```

Comment:
Please use f-strings instead of string concatenation.

Reply:
Fixed in [this ](https://github.com/home-assistant/home-assistant/pull/30445/commits/23b3e5f5c96fd29b03496bc926a03e4a3d167036) commit